Changes related to Cached credentials functionality (#23166)
demisto/content: Packs/DelineaSS/ReleaseNotes/3_0_0.md

#### Integrations
##### Delinea Secret Server

> - Added Feature to provide support to save Credentials into the cache. Now user can access secret server with cached credentials id’s.

Reviewer: ```suggestion
- Added support for saving credentials into the cache. Now user can access secret server with cached credentials id’s.
```

Author: Updated the chnages

> - Implemented the job scheduler to sync the cached credentials in every 30 minutes.

Reviewer: Which job scheduler?
